changelog/_unreleased/2020-09-15-add-support-for-repository-autowiring.md

```diff
@@ -0,0 +1,23 @@
+---
+title:  Add support for autowiring of entity repositories
+issue:  NEXT-10918
+author:             Hendrik Söbbing
+author_email:       hendrik@soebbing.de
+author_github:      @soebbing
+---
+# Core
+* Changed `\Shopware\Core\Framework\DependencyInjection\CompilerPass\EntityCompilerPass` to also register an alias for arguments
+___
+# Upgrade Information
+
+### Entity Repository Autowiring
```

Comment:
Might want to change to `##` headline. Or is this intended?

Reply:
No, will do. Thought that the hierarchy should be that way